src/jvm/main/org/jetbrains/kotlinx/lincheck/strategy/managed/SnapshotTracker.kt

```diff
@@ -237,11 +253,9 @@ class SnapshotTracker {
         )
     }
 
-    private fun getDeclaringClass(obj: Any?, className: String, fieldName: String): Class<*> {
-        return Class.forName(className).let {
-            if (obj != null) it
-            else it.findField(fieldName).declaringClass
-        }
+    private fun getDeclaringClass(obj: Any?, clazz: Class<*>, fieldName: String): Class<*> {
+        return if (obj != null) clazz
```

Comment:
When using both clauses in `if`, please wrap the branches with brackets, it makes the flow more visible when reading the code